### DISCUSSION TITLE
Installation without ISO on ubuntu 18.04

### DISCUSSION THREAD

**Discussion by zackaria-12** [Security-Onion-Solutions/securityonion#7660] [Unsupported Versions]
Hello,
Currently i have done an secuirty onion installation on a ubuntu 18.04 without a ISO. the reason for this is that my virtual machine is hosted on azure. I have already tried with VB as hyper visor but i have seen that nested virtualization brings problems for the VM-network. so i decided to install secuirty onion directly to the VM, but during the setup i get an error and that i should look in the error.logs. if i try to use the so-status command its says that the command is unknown. 

![error](https://user-images.githubusercontent.com/82409606/160456944-1b8b21df-17ba-4430-a83d-d3aa233f2d12.PNG)

**Comment by dougburks** [comment]
so-status is not there because the installation did not complete.

You can look at `/root/sosetup.log` for additional clues.
